File: fixtures/scale/java/medium/src/calc/io/Banner.java
package calc.io;

public class Banner {
    private static final String VERSION = "1.4.2";

    public static String render() {
        StringBuilder sb = new StringBuilder();
        sb.append("mathwork ").append(VERSION).append('\n');
        sb.append("type an expression, :help for commands, :quit to exit");
        return sb.toString();
    }
}

File: fixtures/scale/scala/medium/src/calc/io/Banner.scala
package calc.io

object Banner {
  private val Version = "2.1.0"

  def render: String = {
    val sb = new StringBuilder
    sb.append("mathwork ").append(Version).append('\n')
    sb.append("type an expression, :help for commands, :quit to exit")
    sb.toString
  }
}
